scrapi/harvesters/test_doepages/__init__.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import unicode_literals
```

Comment:
This harvester is just a placeholder, I wanted to see what it would look like to have one. It would be nice to have another abstraction that handled harvesting, but that is a separate problem.